app/presenters/two_factor_authentication/sign_in_selection_presenter.rb

```diff
@@ -0,0 +1,32 @@
+module TwoFactorAuthentication
+  class SignInSelectionPresenter
+    include ActionView::Helpers::TranslationHelper
+
+    attr_reader :configuration, :user
+
+    def initialize(user:, configuration:)
+      @user = user
+      @configuration = configuration
+    end
+
+    def render_in(view_context, &block)
+      view_context.capture(&block)
+    end
+
+    def type
+      method.to_s
+    end
+
+    def label
+      raise "Unsupported login method: #{type}"
+    end
+
+    def info
+      raise "Unsupported login method: #{type}"
+    end
+
+    def disabled?
+      false
+    end
```

Comment:
Are we ever using `disabled?` in a sign-in context?

Reply:
Yes we are, can be seen here https://github.com/18F/identity-idp/blob/main/app/views/two_factor_authentication/options/index.html.erb#L30

Reply:
when phone is out users cant use phone cause of an outage, we disable sign in.

Reply:
Ah, I missed that! Good catch